.github/workflows/ci-coverage-build.yml

```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   coverage:
     name: coverage build
-    uses: ros-controls/ros2_control_ci/.github/workflows/reusable-build-coverage.yml@master
+    uses: ros-controls/ros2_control_ci/.github/workflows/reusable-build-coverage.yml@optional_repos
```

Comment:
```suggestion
    uses: ros-controls/ros2_control_ci/.github/workflows/reusable-build-coverage.yml@master
```

Reply:
once https://github.com/ros-controls/ros2_control_ci/pull/64 is merged

Reply:
Awesome :+1: